doc/nrf/releases/release-notes-1.9.1.rst

```diff
@@ -0,0 +1,122 @@
+In future releases, Segger Embedded Studio will no longer be tested with the build system and it is not recommended for new projects.
+
+nRF Connect extensions for Visual Studio Code will be under continued active development and supported and tested with future |NCS| releases.
+It is recommended for all new projects.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - :ref:`nrfxlib:nrf_modem_changelog`
+   * - LwM2M carrier library
+     - :ref:`liblwm2m_carrier_changelog`
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v1.9.1`_ for the list of issues valid for the latest release.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Bluetooth LE
+------------
+
+*
+
+Thread
+------
+
+* Updated OpenThread libraries with a set of bug fixes from Zephyr upstream.
+  See the `Zephyr`_ section for details.
+
+Trusted Firmware-M
+==================
+
+* Fixed the NCSDK-14015 known issue that would cause crash during boot when the :kconfig:`CONFIG_RPMSG_SERVICE` Kconfig option was enabled on the nRF5340 SoC.
+* Fixed the NCSDK-13949 known issue where the TF-M Secure Image would copy FICR to RAM on nRF9160.
+* Fixed the NCSDK-12306 known issue where a usage fault would be triggered in the debug build on nRF9160.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``45ef0d2``, plus some |NCS| specific additions.
+This is the same commit ID as the one used for |NCS| :ref:`v1.9.0 <ncs_release_notes_190>`.
+
+For a complete list of |NCS| specific commits since v1.9.0, run the following command:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^45ef0d2
+
+Zephyr changes incorporated into |NCS|
+--------------------------------------
+
+This section contains changes in Zephyr that were cherry-picked into |NCS| for this release.
+
+*
```

Comment:
@edmont , please provide input for OpenThread changes.

Reply:
```
Thread
======

* Fixed the KRKNWK-13059 bug where a Thread 1.2 device would occasionally send the wrong Link Layer Frame Counter TLV making the link with the neighbor unusable.
* Fixed the KRKNWK-12923 bug where a Thread 1.2 device would occasionally assert due to wrong transmission error being reported to the MAC layer.
* Fixed the KRKNWK-12862 bug where a Thread device would not properly handle fragmented IPv6 packets.
```

Reply:
Are these fixes cherry-picked from Zephyr? Just want to confirm.

Reply:
Yes.